.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -59,6 +59,22 @@ on:
         default: ''
         required: False
 
+      e2e_binaries_artifact:
+        type: string
+        default: ''
+        required: False
+
+      e2e_testing_mode:
+        type: string
+        default: 'full'
```

Comment:
can we add a `description` describing all the options and what each means?

Reply:
Ok, added